docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '2.1'
+volumes:
+  resin-data: {}
+services:
+  balena-supervisor:
+    build: ./
+    privileged: true
+    tty: true
+    restart: always
+    network_mode: host
+    environment:
+      - DOCKER_ROOT=/mnt/root/var/lib/docker
+      - DOCKER_SOCKET=/var/run/balena-engine.sock
+#       - "BOOT_MOUNTPOINT=$BOOT_MOUNTPOINT"                       <--- Must be set as service variable
```

Comment:
What do you mean by must be set as service variable? A few of these could be set here, e.g. `DELTA_ENDPOINT`, `LISTEN_PORT`

Reply:
I wrote those because from the [startup](https://github.com/balena-os/meta-balena/blob/c681bc3f4759e8f337ace3e5483bfa84d2ed268d/meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor/start-balena-supervisor#L60-L85) script it uses some env vars which means the Supervisor's runtime uses some dynamically set variables that we won't know at build time.

...I just thought of an issue regarding this and will get back to you on FD